taxi/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from taxi_service.settings import AUTH_USER_MODEL
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+    class Meta:
+        ordering = ("name",)
+
+    def __str__(self):
+        return f"{self.name} ({self.country})"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=63)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(AUTH_USER_MODEL)
```

Comment:
add related name for Car model)
dont forget about [checklist.md](https://github.com/mate-academy/py-taxi-service-startproject/blob/master/checklist.md)